website/content/docs/connect/ca/index.mdx

```diff
@@ -4,6 +4,26 @@ page_title: Connect - Certificate Management
 description: An overview of the Connect Certificate Authority mechanisms.
 ---
 
+### CA and Certificate relationship
+
+This diagram shows the relationship between the CA certificates in Consul primary and
+secondary.
+
+![CA relationship](/img/cert-relationship.svg)
```

Comment:
For the image, two suggestions.

- Move the arrow that is covering the Consul primary title
- Add the word datacenter to the boxes. You can change the title in boxes if you want.

Reply:
Last nitpick for the image. The user has to twist their neck to read the vertical text. We can avoid this by placing the text above the grey box.

Reply:
The issue with that is that it will get under an arrow (in the primary DC box)

Reply:
@dhiaayachi You know, we could just use DC1 and DC2 that is a pattern we use throughout the docs. That would address the problem. Plus, you did specify Consul primary and secondary datacenter in the tex

Reply:
Should look better now